tubesync/sync/models.py

```diff
@@ -482,6 +489,15 @@ def type_directory_path(self):
     def make_directory(self):
         return os.makedirs(self.directory_path, exist_ok=True)
 
+    @property
+    def get_thumbnail_url(self):
+        if self.source_type==Source.SOURCE_TYPE_YOUTUBE_PLAYLIST:
+            raise Exception('This source is a playlist so it doesn\'t have thumbnail.')
```

Comment:
Likely this needs nicer error handling. This will bubble up as a 500 probably if someone ticks the tickbox on a playlist.

Reply:
This function is only called if the tickbox is checked AND the source is not a playlist, so it cannot be triggered until it is called from somewhere it should not